runtime/src/accounts.rs

```diff
@@ -208,13 +208,15 @@ impl AccountsDB {
     pub fn add_fork(&self, fork: Fork, parent: Option<Fork>) {
         let mut info = self.fork_info.write().unwrap();
         let mut fork_info = AccountsForkInfo::default();
-        if parent.is_some() {
-            fork_info.parents.push(parent.unwrap());
-            if let Some(list) = info.get(&parent.unwrap()) {
+        if let Some(parent) = parent {
+            fork_info.parents.push(parent);
+            if let Some(list) = info.get(&parent) {
                 fork_info.parents.extend_from_slice(&list.parents);
             }
         }
-        info.insert(fork, fork_info);
+        if let Some(old_fork_info) = info.insert(fork, fork_info) {
+            panic!("dupe forks! {} {:?}", fork, old_fork_info);
```

Comment:
nit: "dupe" => "duplicate".   Full English is more better thn shrt cutz that ppl may nt understd